### ISSUE TITLE
Create an endpoint to set site browser selected folder

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#20190]
**Is your feature request related to a problem? Please describe.**
We need to link to a specific folder in the site browser portlet.

The way DotCMS does that is by setting the selected folder in the session.

**Describe the solution you'd like**
We need an endpoint that we hit with the folder information (not sure if is id, inode or path) and it set that folder in the session.

**Comment by fmontes** [comment]
https://github.com/dotCMS/core/pull/20238

**Comment by fmontes** [comment]
How it works:

```
fetch('/api/v1/browser/selectedfolder', {
  method: 'PUT',
  body: JSON.stringify({ "path":"//newsite.com/application/containers/test" }),
  headers: {
    'Content-type': 'application/json; charset=UTF-8'
  }
})
.then(res => res.json())
.then(console.log)
```

**Comment by fmontes** [comment]
Is not selecting (opening) the last folder we are passing.

**Comment by fmontes** [comment]
https://github.com/dotCMS/core/pull/20274

**Comment by bryanboza** [comment]
We need some work here: 

Based on the postman test: 
- `Get folders and get files` both are returning the same response, also need validate something related to the response, both ones are only validating the response code. 
- `Set selected folder with host` is failing 
- Also in all the 7 test that we have, we need to add more `@test` scenarios, we are validating just response codes here.

**Comment by jdotcms** [comment]
hi @bryanboza 
These services only set things on the session to hack the site browser current path, nothing to return so nothing to test rather than the status code.

As we already talk, the test suppose to running on empty starter. We want to create the {{defaulthost}} but it will be a future feature

**Comment by bryanboza** [comment]
Fixed, tested on release-21.05 // Postgres // FF
